mentors/bash/m.json

```
@@ -1,4 +1,12 @@
 [
+  {
+    "github_username": "machuu",
+    "name": "Matt Hoover",
+    "link_text": "Personal Site",
+    "link_url": "http://hoover.ml",
+    "avatar_url": "http://gravatar.com/avatar/6dcd94787d387411653a88ac408c09dd.png",
+    "bio": "I use BASH daily at work and home, and I think it is a foundational skill for mastering any UNIX-like system."
```

Is it common to have bash written in all caps?
```suggestion
    "bio": "I use Bash daily at work and home, and I think it is a foundational skill for mastering any UNIX-like system."
```

---

I've seen it that way enough times that I started referencing it that way.

After looking at the official README, it should probably be just uppercase B.

Should I update my bio in the Pull Request?